app/forms/idv/how_to_verify_form.rb

```diff
@@ -0,0 +1,41 @@
+module Idv
+  class HowToVerifyForm
+    include ActiveModel::Model
+    ATTRIBUTES = [:selection].freeze
+    REMOTE = 'remote'
+    IPP = 'ipp'
```

Comment:
```suggestion
    REMOTE = 'remote'.freeze
    IPP = 'ipp'.freeze
```

I wouldn't go out of your way at all because this is really trivial, but I think these should technically be frozen like is done in https://github.com/18F/identity-idp/blob/d366044328cf6de17a923008b525c0020d0da8e0/app/forms/security_event_form.rb#L9-L11

But please only deal with this if you're already editing other stuff; this isn't _remotely_ a blocking concern.

Reply:
If you add this magic comment to the top of the file, you don't need to freeze each individual string
```
# frozen_string_literal: true
```
a lot of our files already have it